hack/.custom-gcl.yaml

```diff
@@ -0,0 +1,6 @@
+version: v2.4.0
```

Comment:
Can we build the custom linter without this file ?
I noticed that k/k doesn't add it: https://github.com/kubernetes/kubernetes/commit/2f50d0a510d9508ed736c61662f2a7b4e2636359#diff-409f95e022c4def5d0cab12c8d3a6969db969b6c7ecd1c973e7d2825eb8daa94

We can use tools.go to have this tool as dependency: https://github.com/kubeflow/trainer/blob/master/hack/tools.go

Reply:
Yes, it doesn't seem like it's needed.